### PR TITLE
fix-use-issue138

### PR DESCRIPTION
**Pull request by maamo888**
https://github.com/lvht/phpcd.vim/issues/138

### CODE REVIEW

File: autoload/phpcd.vim [hunk 1]
```diff
@@ -56,7 +56,7 @@ function! phpcd#CompletePHP(findstart, base) " {{{
 
 		let [current_namespace, imports] = phpcd#GetCurrentNameSpace()
 
-		if context =~? '\v^use\s*' " {{{
+		if context =~# '\v^(use$|use+\s.*)' " {{{
```
Comment (lvht) [inline autoload/phpcd.vim]: The `use+\s.*` seems not right. The `+` means **one or more** `e`.

I think `\v^use$` is enough.

Comment (maamo888) [reply autoload/phpcd.vim]: Thanks revied.

It is for use when it is completion in the middle of input of `use`.

Comment (maamo888) [reply autoload/phpcd.vim]: **tmux**

phpcd18031701

Comment (lvht) [reply autoload/phpcd.vim]: Even in the `use ...` case, the `context` will be `use`

Comment (maamo888) [reply autoload/phpcd.vim]: Thanks debug function.

## php

```
<?php
use Lvht\|
```

## :echo phpcd#GetCurrentSymbolWithContext()


```
['', 'use', '\Lvht', {'@': ''}]
```

## \v^use$ snapshot



## \v^(use$|use+\s.*) snapshot

Comment (maamo888) [reply autoload/phpcd.vim]: Thank you for merge.

phpcd.vim great!

**Pull request merged**